tools/apksize/apksize.gradle

```diff
@@ -50,6 +51,35 @@ task uploadMeasurements(type: UploadMeasurementsTask) {
     uploader = "https://storage.googleapis.com/firebase-engprod-metrics/upload_tool.jar"
 }
 
+task generateCoverageMeasurements(type: GenerateCoveragePercentsTask, group: 'Coverage') {
+
+    // TODO(yifany@): Way too cumbersome.
```

Comment:
Can you be more specific on this TODO on how this can be improved?

Reply:
Done.